components/shared/ui/appPage.tsx

```diff
@@ -2,26 +2,27 @@ import { FC, ReactNode } from "react";
 import { Footer } from "./footer";
 import { Menu } from "./menu";
 import { useSmartLink } from "../../../lib/useSmartLink";
-import { createDefaultMenu } from "../../../lib/constants/menu";
 import { ValidCollectionCodename } from "../../../lib/types/perCollection";
 import { SiteCodenameProvider } from "../siteCodenameContext";
 import { createItemSmartLink } from "../../../lib/utils/smartLinkUtils";
+import { Navigation } from "../../../models";
 
 type Props = Readonly<{
   children: ReactNode;
   siteCodename: ValidCollectionCodename;
   itemId?: string;
+  siteMenu?: Navigation;
 }>;
 
 export const AppPage: FC<Props> = props => {
   useSmartLink();
   return (
     <SiteCodenameProvider siteCodename={props.siteCodename}>
       <div className="min-h-full grow flex flex-col items-center overflow-hidden">
-        <Menu links={createDefaultMenu(props.siteCodename)} />
+        {props.siteMenu && <Menu item={props.siteMenu} />}
```

Comment:
A missing menu is most likely a mistake rather than an intentional design. Showing some kind of an error here, in that case, might help editors identify the mistake.